app/api/db/migrations/versions/2022_08_29_14_29_10_b78c1c9029f5_first_migration.py

```diff
@@ -0,0 +1,48 @@
+"""first migration
+
+Revision ID: b78c1c9029f5
+Revises:
+Create Date: 2022-08-29 14:29:10.441061
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "b78c1c9029f5"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "eligibility_screener",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("eligibility_screener_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column("first_name", sa.Text(), nullable=True),
+        sa.Column("last_name", sa.Text(), nullable=True),
+        sa.Column("phone_number", sa.Text(), nullable=True),
+        sa.PrimaryKeyConstraint("eligibility_screener_id"),
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    op.execute("GRANT ALL ON SCHEMA public TO PUBLIC")
```

Comment:
What is this line for? 👀

Reply:
Ah, good catch.

I was attempting to get the admin user created in the DB which was done via a migration, and this was one line of the downgrade I forgot to remove.